survey_phone/__manifest__.py

```diff
@@ -30,19 +29,24 @@
 {
     'name': 'Survey Phone',
     'summary': 'Make the filling of survey by internal users easier.',
-    'version': '10.0.1.1.0',
+    'version': '11.0.0.0.0',
     'category': 'Other',
     'sequence': 150,
     'author': 'Compassion CH',
     'license': 'AGPL-3',
     'website': 'http://www.compassion.ch',
-    'depends': ['survey', 'base_phone'],
+    'depends': ['survey',                       # oca_addons/survey
+                'base_phone',                   # oca_addons/connector-telephony
+                'survey',
+                'partner_contact_birthdate',    # oca_addons/partner_contact
+                'partner_contact_gender',    # oca_addons/partner_contact
```

Comment:
Why did you add those two dependencies?

Reply:
I added them because I needed some fields they were implementing, aka gender and birth-something. If they are in an other module, I will make the required changes

Reply:
Ok, but the last one is conflicting with the `gender` field we added in `advanced_translation` module. So please just change `partner_contact_gender` to `advanced_translation`

Reply:
Ok